src/main/java/org/example/Repository.java

```diff
@@ -62,6 +65,30 @@ public boolean passwordValidator(String password) {
         Pattern pattern = Pattern.compile("^(?=.*[0-9].*[0-9])(?=.*[A-Z])(?=.*[!@£$%^&*()_+=,./?><:|]).{8,}$");
         return pattern.matcher(password).matches();
     }
+
+    /////////////////////////////////////////////////////  POLICY SERVICE
+
+    public int getCustomerId(String username) throws SQLException, IncorrectUsernamePasswordException {
+        ResultSet resultSet = connect.createStatement().
+                executeQuery(String.format(" SELECT UserId FROM usersregistration WHERE user_username = '%s'", username));
+        if (resultSet.next()) {
+            return resultSet.getInt("UserId");
+        }
+        throw new IncorrectUsernamePasswordException("Incorrect username or password");
+    }
+    public List<String> getListPolicyTypes(int customerId) throws SQLException {
+        List<String> ListPolicytypes = new ArrayList<>();
+        ResultSet resultSet = connect.createStatement().executeQuery(String.format(" SELECT P.PolicyType\n" +
```

Comment:
All things to do with policies should be in a separate table to customer. These are 2 separate concepts and should not be present in the same table.


I'd suggest creating a new repository called something like _PolicyRepository_ which should be responsible for all things Policy related

Reply:
Everything related to Policies has been moved to PolicyRepository.